crates/ide_assists/src/handlers/extract_module.rs

```diff
@@ -825,18 +806,14 @@ fn add_change_vis(
     vis: Option<ast::Visibility>,
     node_or_token_opt: Option<syntax::SyntaxElement>,
 ) -> Option<()> {
-    if let Some(vis) = vis {
-        if vis.syntax().text() == "pub" {
-            ted::replace(vis.syntax(), make::visibility_pub_crate().syntax().clone_for_update());
-        }
-    } else {
```

Comment:
Just removing this should be fine right? Keeping `pub(crate)` as the default visibility modifier after extraction sounds correct to me, the issue was mainly talking about `pub` items being extracted having their visibility reduced to `pub(crate)` before which this branch did.